tests/mobly/base_test_test.py

```diff
@@ -421,10 +421,9 @@ def test_something(self):
     actual_record = bt_cls.results.error[0]
     self.assertEqual(actual_record.test_name, self.mock_test_name)
     self.assertEqual(actual_record.details, MSG_EXPECTED_EXCEPTION)
-    self.assertTrue('in setup_test\n    '
-                    'raise Exception(MSG_EXPECTED_EXCEPTION)\n'
-                    'Exception: This is an expected exception.\n' in
-                    actual_record.stacktrace)
+    self.assertIn('in setup_test\n    raise Exception(MSG_EXPECTED_EXCEPTION)'
+                  '\nException: This is an expected exception.\n',
```

Comment:
nit, maybe just me but the previous format of the string seems more readable

Reply:
Changed